chromadb/api/local.py

```diff
@@ -227,11 +232,11 @@ def reset(self):
 
     def _query(
         self,
-        collection_name,
-        query_embeddings,
-        n_results=10,
-        where={},
-        where_document={},
+        collection_name: str,
+        query_embeddings: Embeddings,
+        n_results: int = 10,
+        where: Optional[Where] = {},
```

Comment:
I don't think the Optional type is valid here, as where should not be None and optional is shorthand for Union[..., None]

Reply:
Fixed.
https://github.com/chroma-core/chroma/pull/200/commits/2f6fdd2416debc726e84151936bf4eae0a049fb8